model/ftn/w3iogomd.ftn

```diff
@@ -3747,7 +3746,7 @@
 !
 !
 !/OMPG/!$OMP PARALLEL DO
-!PRIVATE(JSEA,ISEA,FACTOR,FACTOR2,KD,FKD,USSCO,M1)
+!PRIVATE(JSEA,ISEA,FACTOR,KD,FKD,USSCO,MINDIFF,IB)
```

Comment:
I think the second line is just a comment, where it was intented as a clause to the OMP PARALLEL DO in the first line

Reply:
OK @CarstenHansen, I've made that change, added the !/OMPH/!/T1!$OMP lines outlined by @ukmo-ccbunney and my reply to his comment on that, and changed to NEWLNE/OLDLNE to `CHARACTER\*176`. How would you guys like to add the max switches per line limit?